kafka-connect/kafka-connect-runtime/docker/docker-compose.yml

```diff
@@ -0,0 +1,107 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+volumes:
+  data: {}
+
+services:
+  minio:
+    image: minio/minio
+    hostname: minio
+    environment:
+      - MINIO_ROOT_USER=minioadmin
+      - MINIO_ROOT_PASSWORD=minioadmin
+    ports:
+      - 9000:9000
+      - 9001:9001
+    volumes:
+      - data:/data
+    command: server /data --console-address ":9001"
+
+  create-bucket:
+    image: minio/mc
+    depends_on:
+      - minio
+    volumes:
+      - data:/data
+    entrypoint: mc mb /data/bucket
+
+  iceberg:
+    image: tabulario/iceberg-rest
```

Comment:
Should we really add this dependency? 
I am not sure about the future roadmap of this project. 

Can we use some OSS implementation of the catalog?

Reply:
I'm not sure if there is a better alternative but I'm happy to switch if there is one. @danielcweeks do you have concerns relying on this image?

Reply:
This is currently used by a number of projects as a test depending (python,rust,trino) and is fine for now.  We've discussed moving this into the test fixtures in core and I'll work with @Fokko to get that done and we can move the test dependencies separately.

Reply:
Thanks for the update. 

It would be nice to maintain and publish this image under OSS iceberg account 👍

Reply:
+1

Reply:
I can help on the DockerHub "seat" for apache/iceberg (I just did that for the new openserverless podling). Just let me know 😄

Reply:
Yes. 
With that account we can even publish some more test image like Trino, kafaka-connect, Spark etc. 
And we can maintain these images in Iceberg repo test module or as a separate subproject "Iceland - A playground for Iceberg"

Reply:
Thanks @jbonofre!  That was one of the biggest outstanding issues because we don't currently produce any images as a project.

Reply:
@danielcweeks happy to help ! 😄